### DISCUSSION TITLE
Question: ExternalZoneInfo?

### DISCUSSION THREAD

**Discussion by jpom** [mozilla/hubs#4325] [Hubs Cloud Questions] [answered]
Trying to create a Personal Hubs stack, and I've received this same error 3 times now:

![Screen Shot 2021-06-11 at 12 43 28 PM](https://user-images.githubusercontent.com/1765658/121722372-5fca9200-cab3-11eb-875b-01a0d9e55a1a.png)

I registered two domains with Route 53, Transfer lock is Disabled on both. The key pair & the Hubs Cloud deployment are in the same region. 

From the Log events:

> 2021-06-09T20:04:09.402Z	acbd74c8-7e2f-4294-92d2-4eac3c917583	INFO	RESPONSE:
>  { StackId:
>    'arn:aws:cloudformation:us-east-1:364963340270:stack/NCMAkids-Hubs-Peru-2/8ad071b0-c95d-11eb-b3ba-0e72790b6507',
>   RequestId: 'fb9d196d-217a-46ef-a7b3-fcf3bb0db5df',
>   LogicalResourceId: 'ExternalZoneInfo',
>   PhysicalResourceId: 'route53Dependency-ncmakids.org',
>   Status: 'FAILED',
>   Reason:
>    'undefined See details in CloudWatch Log: 2021/06/09/[$LATEST]8478b9cd8701450bbe21f40f9243da79',
>   Data: undefined }

Anyone have any thoughts about what's going wrong here? Thanks.

**Comment by agriggs2** [comment]
+1 
I am seeing this with a brand new account on both us-east-1 and 2 (created on 01/31/2023)

**Comment by cloo** [comment]
Having the same issue with brand new account here.. 

`ExternalZoneInfo`
`Received response status [FAILED] from custom resource. Message returned: undefined See details in CloudWatch Log: 2023/02/01/[$LATEST]73df558aab534262b0d152db3abbfcc5 (RequestId: b5cc1001-c75a-40de-bb83-0831a6b002bc)`

Using my own subdomain for this. Could it cause the problem?

**Reply by gitopandi** [reply]
Hi, i have the same problem with a subdomain:
`
 {
  StackId: 'arn:aws:cloudformation:eu-west-1:575696718627:stack/hubscloud-01/2622aaf0-b446-11ed-8e48-06df9d727739',
  RequestId: 'b9421665-8ad7-4852-9ad0-876bc5a97641',
  LogicalResourceId: 'ExternalZoneInfo',
  PhysicalResourceId: 'route53Dependency-hubscloud.[...].de',
  Status: 'FAILED',
}`

If you have any hints how to solve it, I appreciate your sharing a lot
Andreas

**Comment by rawnsley** [comment, marked as answer]
This is the origin of the error message: https://github.com/mozilla/hubs-ops/blob/master/cloudformation/stack.yaml#L742

According to the source reference its:

> A Lambda function which implements a Custom Resource for Cloud Formation that gets hosted zones by name from Route53.

The log it is referring to is the output from the lambda, which you can find in AWS CloudWatch. For instance, my logs are in a log group called `/aws/lambda/eu-hubscloud-route53-dependency`. There might be more clues there.
